pyta_template.html

```diff
@@ -14,7 +14,158 @@
+    <a href="index.html#E1135">E1135: unsupported-membership-test</a>
+    <a href="index.html#E1136">E1136: unsubscriptable-object</a>
+    </div>
+  </li>
+  <li class="dropdown">
+    <a href="index.html#String checker" class="dropbtn">String checker</a>
+    <div class="dropdown-content">
+    <a href="index.html#E1305">E1305: too-many-format-args</a>
+    <a href="index.html#E1306">E1306: too-few-format-args</a>
+    <a href="index.html#E1310">E1310: bad-str-strip-call</a>
+    <a href="index.html#W1303">W1303: missing-format-argument-key</a>
+    <a href="index.html#W1305">W1305: format-combined-specification</a>
+    </div>
+  </li>
+  <li class="dropdown">
+    <a href="index.html#String Constant checker" class="dropbtn">String Constant checker</a>
+    <div class="dropdown-content">
+    <a href="index.html#W1401">W1401: anomalous-backslash-in-string</a>
+    </div>
+  </li>
+  <li class="dropdown">
+    <a href="index.html#Classes checker" class="dropbtn">Classes checker</a>
+    <div class="dropdown-content">
+    <a href="index.html#R0201">R0201: No self use</a>
+    <a href="index.html#E0202">E0202: Method hidden</a>
+    <a href="index.html#E0203">E0203: Access to member before definition</a>
+    <a href="index.html#E0211">E0211: No method argument</a>
+    <a href="index.html#E0213">E0213: Self as the first argument</a>
+    <a href="index.html#E0239">E0239: Inheriting from a non-class</a>
+    <a href="index.html#E0241">E0241: Duplicate bases</a>
+    <a href="index.html#E0302">E0302: Unexpected special method signature</a>
+    </div>
+  </li>
+  <li class="dropdown">
+    <a href="index.html#Exceptions checker" class="dropbtn">Exceptions checker</a>
+    <div class="dropdown-content">
+    <a href="index.html#E0701">E0701: Bad exception order</a>
+    <a href="index.html#W0702">W0702: Bare exception</a>
+    <a href="index.html#W0703">W0703: Exception is too generic</a>
+    <a href="index.html#W0705">W0705: Duplicate except blocks</a>
+    <a href="index.html#W0711">W0711: Binary op exception</a>
+    </div>
+  </li>
+
+
+  <li class="dropdown">
+    <a href="index.html#Basic checker" class="dropbtn">Basic checker</a>
+    <div class="dropdown-content">
+    <a href="index.html#C0111">C0111: Missing docstring</a>
+    <a href="index.html#C0102">C0102 Blacklisted name</a>
+    <a href="index.html#C0103">C0103 Invalid Name</a>
+    <a href="index.html#C0112">C0112 Empty Docstring</a>
+    <a href="index.html#C0113">C0113 Unneeded not</a>
+    <a href="index.html#C0121">C0121 Singleton comparison</a>
+    <a href="index.html#C0123">C0123 Unidiomatic type check</a>
+    <a href="index.html#E0101">E0101 Return in init</a>
+    <a href="index.html#E0102">E0102 Function redefined</a>
+    <a href="index.html#E0103">E0103 Not in loop</a>
+    <a href="index.html#E0104">E0104 Return outside function</a>
+    <a href="index.html#e0108-duplicate-argument-name-e0108 ">E0108 Duplicate argument name</a>
```

Comment:
This change is not appropriate. The problem is not with the link, it is in how the header is generated (so please fix it elsewhere).

In general, any inconsistency is probably not correct.